firmware/src/moving_light.c

```diff
@@ -30,15 +31,20 @@
 
 // Various constants
 enum { SOFTWARE_VERSION = 0x00000000 };
-enum {PERSONALITY_COUNT = 2};
+enum { PERSONALITY_COUNT = 2 };
+enum { NUMBER_OF_LANGUAGES = 2 };
 
 static const unsigned int LAMP_STRIKE_DELAY = 50000u;
+static const uint32_t ONE_SECOND = 10000;
 static const char DEVICE_MODEL_DESCRIPTION[] = "Ja Rule Moving Light";
 static const char SOFTWARE_LABEL[] = "Alpha";
 static const char DEFAULT_DEVICE_LABEL[] = "Default Label";
 static const char PERSONALITY_DESCRIPTION1[] = "8-bit mode";
 static const char PERSONALITY_DESCRIPTION2[] = "16-bit mode";
 
+static const char LANGUAGE_ENGLISH[] = "en";
```

Comment:
Right now they are only used in this model. If we go beyond that I'll move them to a common file.